docs/contribution.rst

```diff
@@ -0,0 +1,14 @@
+Contribution guidelines
+=======================
+
+Contribution guidelines for the ``catkit2`` package. 
+
+1. Open an issue on GitHub. Describe the nature of the problem, use tags, and
+   assign yourself or others. 
+2. Make a new branch off of the current develop. Use branch names like ``bugfix/nature_of_bug`` or
+   ``feature/nature_of_feature``. 
+3. When you have finished your proposed changes, test in simulation and if
```

Comment:
I would add some stuff from our internal (RMOL) code author guidelines, feel free to edit as you see fit:

3. Open a Draft pull request as soon as possible, even if your work is far away from being ready to be merged. This allows us to communicate through comments on the PR (permanent and visible to everybody) and to directly see the diff with your code when we talk about it.

    3. a) Link your PR to the issue in step 1. This can be done by writing "Fixes #133" in the description of the PR, where 133 is the number of the issue.
    3. b)Write a detailed description in the PR. This helps you define the scope of the PR, tells others what you are working on, and the reviewer what to expect, test and look for. Update it as necessary and as the PR evolves.
    3. c)Assign the correct assignees (probably yourself) in every pull request, add appropriate labels if applicable. The assignee is responsible for finishing the PR. If you're unable to finish it, the PR should be assigned to someone else who will be the person responsible to pulling the PR over the finish line.

4. Do all the work before marking a PR as "ready to review": rebase, run all test, fix errors, update/write documentation and write unit tests where applicable.
5. Rebase your branch regularly if your development takes a while. If you need help with rebasing or have never done it before, please reach out.
6. Request at least one review and iterate with the reviewer on your PR. If you don't know whose review to request, don't hesitate to ask.
7. Answer all review comments and implement requested changes. Make it clear which comments you are punting to later work and create a new issue on the repo for each of these points and/or summarize them in the PR description (or start working on it immediately on a new branch). Let the reviewer know what changes you made for each comment, or if you didn't understand their comments. Ask for a re-review from the reviewer once you are done. Don't mark the comments as resolved; this is the job of the reviewer.
8. Once the PR is approved, it can be merged by anyone, but it never hurts to double-check with someone qui